src/deluge/dsp/delay/delay_buffer.h

```diff
@@ -120,7 +120,7 @@ class DelayBuffer {
 			return;
 		}
 		// If delay buffer spinning above sample rate...
-		if (resample_config_.value().actualSpinRate >= kMaxSampleValue) {
+		if (resample_config_->actualSpinRate >= kMaxSampleValue) {
```

Comment:
Will this change not add a problem in that it's now entirely unchecked?

Reply:
No, it's checked up at the top of the function and early returns if the optional doesn't have a value.

Reply:
Ok perfect